db/migrate/20210520185112_create_orcid_source_records.rb

```diff
@@ -1,7 +1,8 @@
 class CreateOrcidSourceRecords < ActiveRecord::Migration[6.0]
+  # `:text` column with 16MB is the same as MySQL's MEDIUMTEXT but works on sqlite3
   def change
     create_table :orcid_source_records do |t|
-      t.mediumtext :source_data
+      t.text :source_data, limit: 16_777_215
       t.bigint :last_modified_date
```

Comment:
see https://github.com/sul-dlss/sul_pub/blob/main/db/migrate/20160405194617_switch_to_medium_text.rb for a similar choice in a different migration